composer.json

```diff
@@ -69,175 +69,175 @@
+        "google/cloud-network-management": "1.5.0",
+        "google/cloud-network-security": "0.6.0",
+        "google/cloud-notebooks": "0.7.0",
+        "google/cloud-optimization": "0.6.0",
+        "google/cloud-orchestration-airflow": "1.5.0",
+        "google/cloud-org-policy": "0.6.0",
+        "google/cloud-osconfig": "1.3.0",
+        "google/cloud-oslogin": "1.8.0",
+        "google/cloud-policy-troubleshooter": "1.3.0",
+        "google/cloud-policysimulator": "0.2.0",
+        "google/cloud-policytroubleshooter-iam": "0.2.0",
+        "google/cloud-private-catalog": "0.4.0",
+        "google/cloud-profiler": "1.3.0",
+        "google/cloud-pubsub": "1.49.0",
+        "google/cloud-rapidmigrationassessment": "0.3.0",
+        "google/cloud-recaptcha-enterprise": "1.8.0",
+        "google/cloud-recommendations-ai": "0.7.0",
+        "google/cloud-recommender": "1.11.0",
+        "google/cloud-redis": "1.9.0",
+        "google/cloud-redis-cluster": "0.2.0",
+        "google/cloud-resource-manager": "0.8.0",
+        "google/cloud-resource-settings": "1.2.0",
+        "google/cloud-retail": "1.5.0",
+        "google/cloud-run": "0.8.0",
+        "google/cloud-scheduler": "1.10.0",
+        "google/cloud-secret-manager": "1.13.0",
+        "google/cloud-securesourcemanager": "0.2.0",
+        "google/cloud-security-center": "1.21.0",
+        "google/cloud-security-private-ca": "1.5.0",
+        "google/cloud-security-public-ca": "0.3.0",
+        "google/cloud-securitycentermanagement": "0.2.0",
+        "google/cloud-service-control": "1.3.0",
+        "google/cloud-service-directory": "1.3.0",
+        "google/cloud-service-management": "1.3.0",
+        "google/cloud-service-usage": "1.3.0",
+        "google/cloud-shell": "1.3.0",
+        "google/cloud-spanner": "1.68.0",
+        "google/cloud-speech": "1.16.0",
+        "google/cloud-sql-admin": "0.11.0",
+        "google/cloud-storage": "1.36.1",
+        "google/cloud-storage-transfer": "1.4.0",
+        "google/cloud-storageinsights": "0.3.0",
+        "google/cloud-support": "0.2.0",
+        "google/cloud-talent": "1.3.0",
+        "google/cloud-tasks": "1.14.0",
+        "google/cloud-telcoautomation": "0.2.0",
+        "google/cloud-text-to-speech": "1.8.0",
+        "google/cloud-tpu": "1.4.0",
+        "google/cloud-trace": "1.8.0",
+        "google/cloud-translate": "1.17.0",
+        "google/cloud-video-live-stream": "0.7.0",
+        "google/cloud-video-stitcher": "0.8.0",
+        "google/cloud-video-transcoder": "0.10.0",
+        "google/cloud-videointelligence": "1.15.0",
+        "google/cloud-vision": "1.8.0",
+        "google/cloud-vm-migration": "0.6.0",
+        "google/cloud-vmware-engine": "0.4.0",
+        "google/cloud-vpc-access": "1.3.0",
+        "google/cloud-web-risk": "1.5.0",
+        "google/cloud-web-security-scanner": "0.8.0",
+        "google/cloud-workflows": "0.5.0",
+        "google/grafeas": "0.9.0",
+        "google/longrunning": "0.3.0",
```

Comment:
this version bump is causing the build to fail. See https://github.com/googleapis/gax-php/pull/511

Reply:
Fixed in https://github.com/googleapis/gax-php/releases/tag/v1.26.1